content/references/rippled-api/public-rippled-methods/nft-methods/nft_sell_offers.md

```diff
@@ -0,0 +1,112 @@
+---
+html: nft_sell_offers.html
+parent: nft-methods.html
+blurb: Get a list of all sell offers for a NFToken.
+labels:
+  - Non-fungible Tokens, NFTs, NFTokens
+---
```

Comment:
Add `status: not_enabled` to the frontmatter (for all 3 methods) so the page gets the warning flask in the sidebar.